packages/aragon-cli/src/commands/devchain_cmds/utils/ganache-vars.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
**Todo**: Replace 
```js
export default { funcA, funcB }
```
with
```js
export funcA
export funcB
```
**And** make sure other modules importing it are compatible with the new syntax